specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -0,0 +1,16111 @@
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RecoveryWalkResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "VirtualMachineScaleSets_ForceRecoveryServiceFabricPlatformUpdateDomainWalk_MaximumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ForceRecoveryServiceFabricPlatformUpdateDomainWalk_MaximumSet_Gen.json"
+          },
+          "VirtualMachineScaleSets_ForceRecoveryServiceFabricPlatformUpdateDomainWalk_MinimumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ForceRecoveryServiceFabricPlatformUpdateDomainWalk_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/convertToSinglePlacementGroup": {
+      "post": {
+        "tags": [
+          "VirtualMachineScaleSets"
+        ],
+        "operationId": "VirtualMachineScaleSets_ConvertToSinglePlacementGroup",
+        "description": "Converts SinglePlacementGroup property to false for a existing virtual machine scale set.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine scale set to create or update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VMScaleSetConvertToSinglePlacementGroupInput"
+            },
+            "description": "The input object for ConvertToSinglePlacementGroup API."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "x-ms-examples": {
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen.json"
+          },
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/setOrchestrationServiceState": {
+      "post": 
```

Comment:
Please make this `location` required because other `Resource` definiton in other swaggers are required. This is causing the failure in azure-sdk-for-go-track2

Reply:
cannot find the exact spot so I found it: https://github.com/Azure/azure-rest-api-specs/pull/16539/commits/2660ee02e881821c563ef367d6c1d0a15cddab13

This is by design from the CRP team.  Do all the "Resource" definitions have to be the same or is this a warning? 

@chasevanb could you give input on the reason for removing the 'required' tag

Reply:
Spoke to chase and these are approved breaking changes. The 'required' tag for this Location was removed from previous versions in this [PR](https://github.com/Azure/azure-rest-api-specs/pull/15940/files). 
This is to resolve this [bug ](https://msazure.visualstudio.com/One/_workitems/edit/10474356)

Reply:
put back 'required' tag for Resource.Location property and put in new object for VM_List return object